cfpb/cfgov-refresh
# 2020 rural tool

Updates the Rural and Underserved tool for 2020

## Testing

- Visit http://localhost:8000/rural-or-underserved-tool/
- Note 2020 is the default
- Make a query
- Query should return without error

Also, looks like when this was moved into wagtail the data was copied over from the repo but also pushed to S3 from which it's actually served. These files can thus be safely removed.
